tests/general/InvalidLinkBearTest.py

```diff
@@ -3,6 +3,7 @@
 import requests
 import requests_mock
 import unittest
+from unittest.mock import patch, call
```

Comment:
This `call` import is doing bad things when py.test is run with --doctest-modules. Either import straight unittest.mock (probably due to it overriding sys.call() or something like that), or find a way to namespace it differently